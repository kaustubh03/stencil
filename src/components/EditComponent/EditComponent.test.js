import React from "react";
import renderer from "react-test-renderer";
import Editcomponent from ".";
import configureStore from "../../redux/configureStore";

const store = configureStore();
it("Editcomponent: default", () => {
  const component = renderer.create(<Editcomponent store={store} />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
