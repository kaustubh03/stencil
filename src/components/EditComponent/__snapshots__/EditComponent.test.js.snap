// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Editcomponent: default 1`] = `
<div
  className="post"
>
  <form
    className="form"
    onSubmit={[Function]}
  >
    <input
      defaultValue={null}
      placeholder="Enter Post Title"
      required={true}
      type="text"
    />
    <br />
    <br />
    <textarea
      cols="28"
      defaultValue={null}
      placeholder="Enter Post"
      required={true}
      rows="5"
    />
    <br />
    <br />
    <button>
      Update
    </button>
  </form>
</div>
`;
